README.md

```diff
@@ -1 +1,4 @@
-# mle-training
\ No newline at end of file
+# mle-training
+To run the code in nonstandardcode.py :
+1. You need to setup the environment with the environment.yml file provided 
+2. After setting up the environment you can run "python3 nonstandardcode.py" in the console
```

Comment:
Incomplete README. Add the following

1. Project description
2. Command to create environment from env.yml you generated
3. Command to activate environment
4. Command to run Python script
Add the above points in respective sections.

Always use syntax highlighting while specifying commands